### PR TITLE
Fuzzy Opt-Out GA - MERGE 9/9

### PR DESCRIPTION
**Pull request by KellieHawks**
# Pull Request/Issue Resolution

Closes #**ISSUE_NUMBER_HERE**

#### Is this change associated with a Braze feature/product release?
- [ ] Yes (**Insert Feature Release Date Here**)
- [ ] No

---

<details>
<summary>✔️ Pull Request Checklist</summary>
<br>

- [ ] Check that all links work.
- [ ] Ensure you have completed [our Contributors License Agreement](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is **done and ready to be reviewed for merge**. Are you an internal product manager? Reference the internal reviewing chart to tag the appropriate reviewer.
- [ ] Tag others as reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `assets` > `js` > `broken_redirect_list.js`

</details>

<details>
<summary>⭐ Helpful Wiki Shortcuts</summary>
<br>

- [Writing Style Guide](https://docs.google.com/document/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub)
- [Image Style Guide](https://docs.google.com/document/d/e/2PACX-1vRJSkwcjmjrTfLDagZccLpOMMyh5NN5SXRZSjz12cRAHbX4OrUmhvCmYpf_p5YB-9r4_jSOQLkicQIH/pub)
- [Styling Test Page](https://www.braze.com/docs/home/styling_test_page/)

</details>

<details>
<summary>❗ ATTN: For PR Reviewers</summary>
<br>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Read our [Previewing Documentation page](https://github.com/braze-inc/braze-docs/wiki/Previewing-and-Testing-Documentation) to see how to check the deployment.
  - [ ] Preview all changes in the linked Vercel environment by clicking the preview link in the vercel-bot comment in your PR.
</details>

<details>
<summary>❗ ATTN: Internal Reviewing Chart </summary>
<br>
<b>Work at Braze and not sure who to tag for review?</b> <br>Before tagging @timothy-kim or @KellieHawks for a general review, reference the following chart to see if a specific product vertical/reviewer applies to your pull request.
<br><br>
<table>
<tr>
    <td><b>Reviewer</b></td>
    <td><b>Product Vertical</b></td>
  </tr>
  <tr>
    <td>@Timothy-Kim</td>
    <td>Data Infrastructure<br>Platform Infrastructure<br>Quality Infrastructure</td>
  </tr>
  <tr>
    <td>@kelliehawks</td>
    <td>Internal Tools<br>Product Partnerships<br>SDKs<br>SMS</td>
  </tr>
  <tr>
    <td>@bre-fitzgerald</td>
    <td>Channels<br>FIX<br>Intelligence<br>Reporting<br>SMB</td>
  </tr>
  <tr>
    <td>@lydia-xie</td>
    <td>Email (Composition and Infrastructure)<br>Ingestion<br>Messaging</td>
  </tr>
</table>
</details>

### CODE REVIEW

File: _docs/_user_guide/message_building_by_channel/sms/keywords/fuzzy_opt_out.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+channel:
+  - SMS
+page_order: 1.1
+---
+
+# Fuzzy opt-out
+
+![][1]{: style="float:right;max-width:30%;margin-left:15px;"}
+
+Users that send SMS with Braze must adhere to the applicable laws, regulations, and industry standards that are defined. For opt-out, the laws dictate that when a user texts STOP that all subsequent messaging related to that messaging program be stopped. Braze automatically processes these messages and unsubscribes the user.
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/sms/keywords/fuzzy_opt_out.md:15]:
```suggestion
Users that send SMS with Braze must adhere to the applicable laws, regulations, and industry standards that are defined. For opt-out, the laws dictate that when a user texts "STOP" that all subsequent messaging related to that messaging program will be stopped. Braze automatically processes these messages and unsubscribes the user.
```

File: _docs/_user_guide/message_building_by_channel/sms/keywords/optin_optout.md [hunk 2]
```diff
@@ -30,4 +30,8 @@ Only the **exact, single-word message** will be processed (case _insensitive_).
 
 If a recipient uses the keywords `HELP` or `INFO`, a response will be triggered automatically. The SMS template for these automatic response messages will be set during your [onboarding][oblink] and phone number procurement period. Note that you may continue to update these responses after the initial onboarding period.
 
+{% alert tip %}
+Interested in expanding your opt-out processing? Try [fuzzy opt-out]({{site.baseurl}}/user_guide/message_building_by_channel/sms/keywords/fuzzy_opt_out/), a feature that attempts to recognise when an inbound message does not match an opt-out keyword, but indicates opt-out intent.
```
Comment (lydia-xie) [inline _docs/_user_guide/message_building_by_channel/sms/keywords/optin_optout.md:34]:
```suggestion
Interested in expanding your opt-out processing? Try [fuzzy opt-out]({{site.baseurl}}/user_guide/message_building_by_channel/sms/keywords/fuzzy_opt_out/), a feature that attempts to recognize when an inbound message does not match an opt-out keyword, but indicates opt-out intent.
```

**Review by lydia-xie — CHANGES_REQUESTED** [review]
looks great! left a couple minor suggestions 👍

**Review by lydia-xie — APPROVED** [review]

File: _docs/_user_guide/message_building_by_channel/sms/keywords/fuzzy_opt_out.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+The criteria for an inbound response to be deemed as "fuzzy" are as follows:
+- If switching a letter with the letter one to the left or right of it on a QWERTY keyword yields a matching opt-out keyword.
+- A substring of the message matches an opt-out keyword.
+
+For example, "Stpo" or "Please stopppp" will be deemed fuzzy and a fuzzy opt-out response will be sent.
+
+## Configure fuzzy opt-out
+
+To configure fuzzy opt-out, navigate to the **Subscription Group Keyword Management** page. 
+1. Under **Subscription Groups**, open the **SMS** section.
```
Comment (bre-fitzgerald) [inline _docs/_user_guide/message_building_by_channel/sms/keywords/fuzzy_opt_out.md:32]: @KellieHawks for reference, these are the lines that confused me. Is opening the SMS section clicking on an SMS subscription group in the **Subscription Groups** section?

 
 The following line as well, "select the opt-out category by selecting the pencil icon", is that the **Edit** button?

**Pull request merged**